report/Sprint_1/Application.tex

```diff
@@ -0,0 +1,92 @@
+\section{Application}
```

Comment:
Kunne godt tænke mig en introduktion til afsnittet, bare så det er ensformet for alle afsnit

Reply:
Der er nu skrevet en kort introduktion